test/har/.eslintrc

```diff
@@ -1,5 +1,2 @@
 env:
     mocha: true
-
-extends:
```

Comment:
These inherit the root .eslintrc so it's not needed.

On Wed, Oct 19, 2016 at 5:35 AM, Jonathan Lee <notifications@github.com>
wrote:

> In test/har/.eslintrc
> <https://github.com/sitespeedio/coach/pull/132#discussion_r83988549>:
>
> > @@ -1,5 +1,2 @@
> >  env:
> >      mocha: true
> > -
> > -extends:
>
> @XhmikosR <https://github.com/XhmikosR> is removing these ok since the
> .eslintrc at the root of the project has the extend? Sorry not super
> familiar with eslint yet and why these are not needed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sitespeedio/coach/pull/132/files/6ccb375e9e19a39b4dbb209819bd0fc08d3400d8#r83988549>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVta2PXwbYMg-OdApH4DmWaZzKahIMks5q1YHWgaJpZM4IxByp>
> .
>